extensions/amp-a4a/0.1/a4a-variable-source.js

```diff
@@ -23,7 +23,7 @@ import {
 } from '../../../src/service/variable-source';
 import {user, userAssert} from '../../../src/log';
 
-const WHITELISTED_VARIABLES = [
+const ALLOWED_VARIABLES = [
```

Comment:
In other locations we've used `ALLOWLISTED_...`, could we be consistent?

Reply:
done